### ISSUE TITLE
spacy_tokenize() does not work with a quanteda corpus

### ISSUE THREAD

**Issue by stefan-mueller** [quanteda/spacyr#175]
The documentation of `spacy_tokenize()` states that "`x` can be a character object, a **quanteda** corpus, or a TIF-compliant corpus data.frame". Yet, it does not seem to work on a **quanteda** corpus, but only on `texts(corpus)`.  Either the behaviour should change or the documentation should be refined.

```r
library(quanteda)
#> Package version: 1.5.1
#> Parallel computing: 2 of 12 threads used.
#> See https://quanteda.io for tutorials and examples.
#> 
#> Attaching package: 'quanteda'
#> The following object is masked from 'package:utils':
#> 
#>     View
library(spacyr)

spacy_initialize(model = "en")
#> Found 'spacy_condaenv'. spacyr will use this environment
#> successfully initialized (spaCy Version: 2.1.4, language model: en)
#> (python options: type = "condaenv", value = "spacy_condaenv")

toks1 <- spacy_tokenize(data_corpus_inaugural, output = "data.frame")
#> Error in UseMethod("spacy_tokenize"): 
#> no applicable method for 'spacy_tokenize' applied to an object of class "c('corpus', 'list')"

toks2 <- spacy_tokenize(texts(data_corpus_inaugural), output = "data.frame")

head(toks2, 10)
#>             doc_id    token
#> 1  1789-Washington   Fellow
#> 2  1789-Washington        -
#> 3  1789-Washington Citizens
#> 4  1789-Washington       of
#> 5  1789-Washington      the
#> 6  1789-Washington   Senate
#> 7  1789-Washington      and
#> 8  1789-Washington       of
#> 9  1789-Washington      the
#> 10 1789-Washington    House
```

**Comment by kbenoit** [comment]
Good point, we just need a method for a **quanteda** corpus. This should be added here:
https://github.com/quanteda/quanteda/blob/master/R/spacyr-methods.R#L98-L101

**Comment by CharlotteAlexander** [comment]
Hello,
I encountered the same error noted above when trying to use spacy_tokenize on a quanteda corpus. I successfully used the workaround mentioned above -- enclosing the corpus name with texts().

However, is there a way to retain docvars that were originally attached to my corpus after tokenization? When I tokenize by sentences and my output is a dataframe, I'm only left with doc_id and tokens columns, having lost all associated docvars.

Below is my code:

my_df <- spacy_tokenize(texts(my_corpus), 
                                      what = "sentence", output = "data.frame") 

I'd like to be able to add something like "use_docvars = TRUE" that is available elsewhere in quanteda.

Thanks!
Charlotte
